src/client/scripts/cardModal.js

```diff
@@ -0,0 +1,72 @@
+"use strict";
+/* global Cards:false SnackbarMethods:false */
+
+Template.cardModal.helpers({
+    cardModalInfo: function(_id) {
+        return Cards.findOne({"_id": _id});
+    }
+});
+
+Template.cardModal.events({
+    "click #submitCommentButton": function(event){//eslint-disable-line
+        event.preventDefault();
+        var comment = event.target.form[0].value;
+        var image = null;
+
+        if(!comment || comment.length <= 4)
+            SnackbarMethods.DisplayMessage("Enter a more valuable comment",500);
```

Comment:
Nice business-y wording lol, also, that's 500ms.... not sure if you only want it to stay for only that long (flashes up and goes away immediately), would probably try to stay to a min of 1-2 secs, I've been doing 3 secs for non-important, and would say 5 or 10 for more important